src/olympia/constants/permissions.py

```diff
@@ -0,0 +1,57 @@
+from collections import namedtuple
+
+
+AclPermission = namedtuple('AclPermission', 'app, action')
+

```

Comment:
Improvement idea: Allow a `help_text` to be set (all the comments for every constant) so that we are able to show them in the admin at some point?